### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by sbauch**
@mike6838 there were some issues with the stuff you committed.

I put some branch protection rules in place, I don't want to be committing to main things that break.

One issue is filenames - everything is case sensitive, so renaming `okta.md` to `Okta.md` broke the build.

Other things below

### CODE REVIEW

File: docs/idp/Okta.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+---
+id: okta
+title: Okta
+---
```
Comment (sbauch) [inline docs/idp/Okta.md:4]: this is called "frontmatter"

see https://v2.docusaurus.io/docs/2.0.0-alpha.65/docs-introduction

File: docs/idp/Okta.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+
+Okta is a publicly traded identity and access management company based in San Francisco. It provides cloud software that helps companies manage and secure user authentication into modern applications, and for developers to build identity controls into applications, website web services and devices.
+
+
```
Comment (sbauch) [inline docs/idp/Okta.md:8]: generally im not sure that just replicating our docs is really valuable. I think we could drop a PDF in there to show an example, but lets generally think through an information architecture that provides value here.

what would someone in CS want to know about an IDP? Sales?

Comment (mike6838) [reply docs/idp/Okta.md:8]: I do want a place where if someone is at a loss to get started they can always appeal to the documentation to move them along. Emails get forwarded and attachments get overlooked, I'd like to give people the chance to put themselves on the right path. The Identity Provdiers > Overview will be more along the lines of what you're thinking, I believe

Comment (sbauch) [reply docs/idp/Okta.md:8]: sure, that's why i think we can drop the actual PDF in. but note that we aren't set up to interpolate your data into here, so this wont help someone actually get set up

File: sidebars.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ module.exports = {
         {
           type: 'category',
           label: 'Identity Providers',
-          items: ['idp/overview', 'idp/okta',]
+          items: ['idp/overview', 'idp/okta', 'idp/azure-active-directory']
```
Comment (sbauch) [inline sidebars.js:29]:
```suggestion
          items: ['idp/overview', 'idp/azure-active-directory', 'idp/okta']
```

**Comment by mike6838** [comment]
Looks good

**Pull request merged**